### ISSUE TITLE
Remove file path from status bar

### ISSUE THREAD

**Issue by SpyTec** [asvetliakov/vscode-neovim#291]
Having the file path in the status bar is a bit much, is it possible to remove this somehow?

**Comment by iMarv** [comment]
Right click the status bar and uncheck Neo Vim.
You won't have a statusbar indicator for the mode your are in though.

**Comment by SpyTec** [comment]
Yep, know about that part. Though as you say, will miss out on everything else

**Comment by Shatur95** [comment]
> set shortmess+=F

Setting will not works since extension hardcodes [shortmess](https://github.com/asvetliakov/vscode-neovim/blob/8b78a7edd7dcee928db9fb036553852c238e8c1a/vim/vscode-options.vim#L3).

The option already exist in `shortmess`, this just not works because you do not need to set it as a string.

**Comment by asvetliakov** [comment]
@Shatur95 You seems better than me in vimscript 😄  care to send a PR ?

**Comment by Shatur95** [comment]
Sure :)
